python/ray/tests/test_actor_failures.py

```diff
@@ -376,6 +376,7 @@ class RestartableActor:
         """An actor that will be reconstructed at most once."""
 
         def echo(self, value):
+            time.sleep(0.1)
```

Comment:
let's make this deterministic using a `SignalActor` instead

Reply:
maybe https://github.com/ray-project/ray/blob/8af8dae561b54dea5fdc76b6eda79b6375741496/python/ray/tests/test_unavailable_actors.py#L137